frontend/src/scenes/billing/PurchaseCreditsModal.tsx

```diff
@@ -0,0 +1,117 @@
+                        })}{' '}
+                        per month. The minimum purchase size for credits is $6,000 per year.
+                    </p>
+
+                    <p className="mb-1 text-md font-semibold">Breakdown</p>
+                    <p>
+                        Due today: <b>${Math.round(+creditForm.creditInput).toLocaleString('en-US')}</b>
+                        <br />
+                        Discount: <b>{creditDiscount * 100}%</b>
+                        <br />
+                        Total credits:{' '}
+                        <b>
+                            $
+                            {(+creditForm.creditInput / (1 - creditDiscount)).toLocaleString('en-US', {
+                                minimumFractionDigits: 2,
+                                maximumFractionDigits: 2,
+                            })}
+                        </b>
+                        <br />
+                        Monthly credits:{' '}
+                        <b>
+                            $
+                            {(+creditForm.creditInput / (1 - creditDiscount) / 12).toLocaleString('en-US', {
+                                minimumFractionDigits: 2,
+                                maximumFractionDigits: 2,
+                            })}
+                        </b>
+                    </p>
+
+                    <LemonField name="creditInput">
+                        {({ value, onChange, error }) => (
+                            <div className="max-w-40">
+                                <LemonInput
+                                    type="number"
+                                    fullWidth={false}
+                                    status={error ? 'danger' : 'default'}
+                                    value={value}
+                                    data-attr="credit-input"
+                                    onChange={onChange}
+                                    prefix={<b>$</b>}
+                                    min={0}
+                                    step={10}
+                                    suffix={<>/ year</>}
+                                    size="small"
+                                />
+                            </div>
+                        )}
+                    </LemonField>
+
+                    <LemonDivider />
+                    <p className="mb-1 text-md font-semibold">Invoice details</p>
+                    <p className="mb-0">
+                        We can either charge your card on file now or send you an invoice. Check the box below if you'd
+                        like to receive an invoice. We will also close any existing open invoices.
+                    </p>
+                    <LemonField name="sendInvoice">
+                        {({ value, onChange }) => (
+                            <LemonCheckbox label="Send me an invoice" checked={value} onChange={onChange} />
+                        )}
+                    </LemonField>
```

Comment:
I'd get rid of all this text and just use a radio button:
- Charge my CC on file (ending in XXXX)
- Send an invoice to x@y.com (maybe info icon with popover that says we'll close open invoices and credits will be applied upon payment)